board/power_saving.h

```diff
@@ -10,33 +10,33 @@ void set_power_save_state(int state) {
     bool enable = false;
     if (state == POWER_SAVE_STATUS_ENABLED) {
       puts("enable power savings\n");
-      if (is_grey_panda) {
+      if ((hw_type == HW_TYPE_GREY_PANDA) || (hw_type == HW_TYPE_BLACK_PANDA)) {
```

Comment:
probably worth defining
` 
has_gps_receiver = (hw_type == HW_TYPE_GREY_PANDA) || (hw_type == HW_TYPE_BLACK_PANDA)
`

Reply:
indeed, `is_grey_panda` should be totally removed from the code.

Reply:
I agree. Added it as a function to board.h